app/views/lesson_plans/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>Lesson Planner</h1>
+
+<% if @lesson_plan_lessons.empty? %>
+  <p>There's nothing here! <%= link_to "Browse our training topics ", topics_path %> to start adding lessons to your plan.</p>
+<% else %>
+  <p>Here are your selected lessons. They're remixable! You can click remove to take a lesson off your lesson plan.</p>
+
+  Share this lesson plan: <input type="text" value="<%= lesson_plan_url @lesson_plan %>">
```

Comment:
We should generate a `SecureRandom.uuid` to use as the url parameter, ultimately. The info isn't terribly private but I think it'd still be weird if you were able to see other lesson plans by incrementing the id.